_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
 [source,hcl]
 ----
 provider "aws" {
+  # The AWS region in which all resources will be created
   region = var.aws_region
+
+  # Require a 3.x version of the AWS provider
+  version = "~> 3.40"
+
+  # Only these AWS Account IDs may be operated on by this template
+  allowed_account_ids = [var.aws_account_id]
+}
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt or via a backend.hcl file. See
+  # https://www.terraform.io/docs/backends/config.html#partial-configuration
+  backend "s3" {}
+
+  # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
+  # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
+  required_version = "= 0.14.8"
 }
 ----
 
-Now use the `iam-password-policy` compliance module from the Gruntwork `terraform-aws-cis-service-catalog` repository, making sure to
-replace the `<VERSION>` placeholder with the latest version from the
-https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases[releases page]:
+Next, use the `account-baseline-root` module from the Gruntwork Infrastructure as Code Library:
 
-.infrastructure-modules/security/iam-password-policy/main.tf
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
 [source,hcl]
 ----
-module "iam_password_policy" {
-  # Make sure to replace <VERSION> in this URL with the latest terraform-aws-cis-service-catalog release
-  source                  = "git@github.com:gruntwork-io/terraform-aws-cis-service-catalog.git//modules/iam-password-policy?ref=<VERSION>"
-  minimum_password_length = var.minimum_password_length
-  aws_region              = var.aws_region
+module "root_baseline" {
+  source = "git::git@github.com:gruntwork-io/terraform-aws-cis-service-catalog.git//modules/landingzone/account-baseline-root?ref=v0.44.10"
+
+  aws_account_id = var.aws_account_id
+  aws_region     = var.aws_region
+  name_prefix    = var.name_prefix
+
+  # If you're running the example against an account that doesn't have AWS Organization created, change the following value to true
+  create_organization = var.create_organization
+
+  # The child accounts to create
+  child_accounts = var.child_accounts
+
+  # IAM users to create in this account
+  users = var.users
+
+  # These are variables you only need to set at test time so that everything can be deleted cleanly.
+  # You will likely NOT need to set this in any real environments.
+  force_destroy_users      = var.force_destroy
+  cloudtrail_force_destroy = var.force_destroy
+  config_force_destroy     = var.force_destroy
+
+  # Enable IAM Access Analyzer
+  iam_access_analyzer_type   = var.iam_access_analyzer_type
+  iam_access_analyzer_name   = var.iam_access_analyzer_name
+  enable_iam_access_analyzer = var.enable_iam_access_analyzer
```

Comment:
This is taken from the `cis-repo/examples/ladningzone/account-baseline-root`.
I believe the IAM Access Analyzer should not be configurable:

```suggestion
  # Enable IAM Access Analyzer
  iam_access_analyzer_type   = var.iam_access_analyzer_type
  iam_access_analyzer_name   = var.iam_access_analyzer_name
```

Reply:
Yeah agreed, I imagine @marinalimeira probably hasn't updated her example yet.

Reply:
I'll follow up and leave a comment on the `root` PR

Reply:
this was already covered in the root PR.